src/main/java/io/supertokens/storage/mysql/config/MySQLConfig.java

```diff
@@ -68,6 +68,15 @@
     @JsonProperty
     private String mysql_thirdparty_users_table_name = null;
 
+    @JsonProperty
+    private String passwordless_users_table = null;
+
+    @JsonProperty
+    private String passwordless_device_table = null;
+
+    @JsonProperty
+    private String passwordless_codes_table = null;
+
```

Comment:
These configs are not needed. They were there for older tables, but we stopped adding them for newer ones since someone can use the prefix config to change table names anyway

Reply:
roger that